components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam5_resus_mom_cnst_no3_oh_mosaic_vbs_tag.in

```diff
@@ -145,6 +135,16 @@ Solution Classes
    cl_a1,   cl_a2,   cl_a3
    mom_a1,  mom_a2,  mom_a3,  mom_a4
    num_a1,  num_a2,  num_a3,  num_a4, num_a5
+ End Explicit
+
+ Implicit
+   O3, OH, HO2, H2O2, CH2O, CH3O2, CH3OOH
+   NO, NO2, NO3, N2O5, HNO3, HO2NO2, PAN
+   C2H5O2, C2H5OOH, CH3CHO, CH3CO3, C2H4, ROHO2
+   ISOP, ISOPO2, C10H16, MVKMACR, MVKO2, NH3, HCL
+   soa_a1,  soa_a2,  soa_a3
```

Comment:
I wonder why the SOA aerosol species are treated differently from the other aerosols.

Reply:
Only soa aerosol has photolysis
 [jsoa_a1->,.0004*jno2]  soa_a1 + hv ->
 [jsoa_a2->,.0004*jno2]  soa_a2 + hv ->
 [jsoa_a3->,.0004*jno2]  soa_a3 + hv ->

Other aerosols don't have chemistry reactions, so put in the explicit solver to save computation time.
If the photolysis is slow, we may also move soa aerosols to the explicit solver. 
@tangq is that right?

Reply:
> I wonder why the SOA aerosol species are treated differently from the other aerosols.

@hwangacme , this is because soa_a[1-3] have photolysis reactions. But I am not sure how fast these photolysis reactions are. @mkshriva , do you think we can move soa_a[1-3] to explicit?